src/material/core/density/_index.scss

```diff
@@ -0,0 +1,64 @@
+//
```

Comment:
Do we have any way to make sure users won't use these functions or variables?

Reply:
I prefixed all variables/functions with an underscore. This should be sufficient? We did the same for the actual density mixins for components. I'll also go through the entire branch again and check if there is more stuff that we need to prefix (in a follow-up).